ansible/roles/fanout/templates/sonic_deploy_202405.j2

```diff
@@ -18,6 +18,16 @@
             "mtu": "9100",
     {% if fanout_hwsku == "Arista-720DT-G48S4" and (fanout_port_config[port_name]['lanes'] | int) <= 24 %}
             "autoneg": "on",
+    {% else %}
+        {% if port_name in device_conn[inventory_hostname] and 'autoneg' in device_conn[inventory_hostname][port_name] %}
+            "autoneg": "{{ device_conn[inventory_hostname][port_name]['autoneg'] }}",
+        {% else %}
+            {% if fanout_hwsku == "Mellanox-SN5600-C256S1" %}
```

Comment:
maybe better to check against platform, since it is needed for all SP3 and SP4 platforms.